packages/app-mobile/components/screens/Notes.tsx

```diff
@@ -246,7 +246,7 @@ class NotesScreenComponent extends BaseScreenComponent<any> {
 				}
 				return buttonFolderId;
 			};
-			if (addFolderNoteButtons && this.props.folders.length > 0) {
+			if (addFolderNoteButtons && this.props.folders.length > 0 && this.props.activeFolderId !== getTrashFolderId()) {
```

Comment:
That would fail for notebooks that are within the trash. Please use `itemIsInTrash` but be aware of this bug: https://github.com/laurent22/joplin/pull/10201#discussion_r1542091597

Reply:
ok I tried fixing the bug #10191
here's the output 

https://github.com/laurent22/joplin/assets/92685826/a41366df-e55c-4a44-852c-41a2ca59ffd8